src/RequestSender.ts

```diff
@@ -1,19 +1,22 @@
-import _Error = require('./Error');
-import utils = require('./utils');
-const {
+import {
   StripeAPIError,
   StripeAuthenticationError,
   StripeConnectionError,
   StripeError,
   StripePermissionError,
   StripeRateLimitError,
-} = _Error;
-
-const {HttpClient} = require('./net/HttpClient');
+} from './Error.js';
+import {
+  emitWarning,
+  normalizeHeaders,
+  removeNullish,
+  stringifyRequestData,
+} from './utils.js';
```

Comment:
Why is there `js` extension everywhere? Shouldn't it work without?

Reply:
It does work without at the moment, but when we add `type: "module"` to our package.json later we'll need this because [file extensions are mandatory in ESM](https://nodejs.org/api/esm.html#esm_mandatory_file_extensions), and the TypeScript compiler will also [need these explicit .js extensions](https://github.com/nodejs/node/issues/46006) when targeting ESM. However now that I think about it, we should add these extensions later in a separate PR when we actually need them.

Reply:
Can the extension be `.ts`?

Reply:
[No.](https://github.com/microsoft/TypeScript/issues/16577#issuecomment-754941937)